scripts/validate_min_versions_in_sync.py

```diff
@@ -28,7 +31,13 @@
 )
 CODE_PATH = pathlib.Path("pandas/compat/_optional.py").resolve()
 SETUP_PATH = pathlib.Path("pyproject.toml").resolve()
+YAML_PATH = pathlib.Path("ci/deps")
+ENV_PATH = pathlib.Path("environment.yml")
 EXCLUDE_DEPS = {"tzdata", "blosc"}
+EXCLUSION_LIST = {
```

Comment:
I think we may need to exclude pyarrow too, as there's a substitution that happens here:

https://github.com/phofl/pandas/blob/18c43651b0497b5e93fd02e4f0bf6d254a25c316/.github/actions/setup-conda/action.yml#L21